enterprise-suite/scripts/install-es.sh

```diff
@@ -46,17 +45,12 @@ function usage() {
 }
 
 # Create arg for the helm install/upgrade lines.  $1 is username.  $2 is password.
-# If not using Travis, we stash credentials in a file.  Seems not to work with Travis.
-# This prevents the credentials being written to a log.  (Travis obfuscates the values.)
+# This prevents the credentials being written to a log.
 function set_credentials_arg() {
-    if [ -z "$TRAVIS" ] ; then
-        CREDS=$(mktemp -t creds.XXXXXX)
-        # write creds to file for use by helm
-        printf '%s\n' "imageCredentials.username: $1" "imageCredentials.password: $2" >"$CREDS"
-        HELM_CREDENTIALS_ARG="--values $CREDS"
-    else
-        HELM_CREDENTIALS_ARG="--set imageCredentials.username=$1,imageCredentials.password=$2"
-    fi
+    CREDS=$(mktemp -t creds.XXXXXX)
+    # write creds to file for use by helm
+    printf '%s\n' "imageCredentials:" "   username: $1" "   password: $2" >"$CREDS"
```

Comment:
YAML is whitespae sensitive, correct? I see 3 spaces instead of 2. My local was 2..not sure if it'll make a difference

Reply:
2 is apparently recommended but 3 etc. will work.